lib/repo/schema.ex

```diff
@@ -1,8 +1,53 @@
 defmodule ExAudit.Schema do
-  def insert_all(module, name, schema_or_source, entries, opts) do
-    # TODO!
-    opts = augment_opts(opts)
-    Ecto.Repo.Schema.insert_all(module, name, schema_or_source, entries, opts)
+  def insert_all(module, name, schema, entries, opts) do
+    opts_with_return =
+      opts
+      |> augment_opts()
+      |> Keyword.put(:returning, true)
```

Comment:
This won't work with MyXQL Adapters since MySQL/MariaDB doesn't have a returning option.

Reply:
Nice catch, will change it to handle those adapters that do not have return option

Reply:
@narrowtux it is update to skip the `insert_all` schema track when the repo is not using `Postgres Adapter`